### PR TITLE
[ADD] SLIP-132 switch for PDF backup key format

### PR DESCRIPTION
**Pull request by maxidev**
Added the SLIP-132 switch on the backup PDF view so the document can properly reflect master key format.
image

### CODE REVIEW

**Comment by ben-kaufman** [comment]
Hi, thanks, looks good for existing wallets, but breaks the popup of the new wallet since there's no element with that id there.
https://github.com/cryptoadvance/specter-desktop/blob/cf6a2b9444ff29c45d02b1ad1e3e93a9f6f0a269/src/cryptoadvance/specter/templates/wallet/receive/wallet_receive.jinja#L179

I guess you can either add the same toggle option there or just put it on false by default if that complicates the UI

**Comment by maxidev** [comment]
PR updated to fix the new wallet creation view, also adding the switch.

File: src/cryptoadvance/specter/templates/wallet/receive/wallet_receive.jinja [hunk 1]
```diff
@@ -217,6 +225,11 @@
 				btn.innerText = 'Hide past addresses';
 			}
 		}
+		function toggleKeyDisplay() {
+			let slip132Checkbox = document.getElementById('use_slip_132_pdf');
+			let xpubs = document.getElementsByClassName('slip132_xpub');
+			let zpubs = document.getElementsByClassName('normal_xpub');
+		}
```
Comment (ben-kaufman) [inline src/cryptoadvance/specter/templates/wallet/receive/wallet_receive.jinja:232]: What is this for? Looks like it doesn't do anything...

Comment (maxidev) [reply src/cryptoadvance/specter/templates/wallet/receive/wallet_receive.jinja]: Done, commented out the selectors and left a minimal detail on why they are not completely removed.

**Review by ben-kaufman — APPROVED** [review]
Looks good, thanks!

**Pull request merged**